cpp/include/cudf/rolling.hpp

```diff
@@ -379,6 +381,125 @@ std::unique_ptr<column> grouped_time_range_rolling_window(
   std::unique_ptr<aggregation> const& aggr,
```

Comment:
Not part of this PR, but this copydoc block seems incorrect.  `preceding_window_in_days` and `following_window_in_days` are a different type.

Reply:
I thought this might be acceptable, since the description still fits. All that changes between the two is that the second supports `UNBOUNDED` window bounds.
I was trying to avoid having to repeat the detailed description for each function. I can try redoing this.

Reply:
I've switched to using `@copydetails`, and redone the parameter descriptions.